moonbeam-types-bundle/index.ts

```diff
@@ -336,6 +337,48 @@ const TYPES_800_undefined: RegistryTypes = {
   MultiLocation: "MultiLocationV1",
 };
 
+const TYPES_900_undefined_deprecated: RegistryTypes = {
+  ...TYPES_800_899,
+  MoonbaseRuntimeAssetRegistrarMetadata: {
+    name: "Vec<u8>",
+    symbol: "Vec<u8>",
+    decimals: "u8",
+    isFrozen: "bool",
+  },
+  PalletCrowdloanRewardsRewardInfo: {
+    total_reward: "Balance",
+    claimed_reward: "Balance",
+    contributed_relay_addresses: "Vec<RelayChainAccountId>",
+  },
+  ParachainStakingNominatorStatus: {
+    _enum: { Active: "Null", Leaving: "RoundIndex" },
+  },
```

Comment:
Do we need to include this one? It doesn't have snake case. `_enum` is special syntax to encode rust-style enum variants.

Reply:
My bad, fixing it